docs/content/querying/filters.md

```diff
@@ -270,3 +241,38 @@ Now suppose that the lower bound is `110` the filter will not match (`110 < 10K`
 |type|This String should always be "contains".|yes|
 |value|A String value to run the search over.|yes|
 |caseSensitive|Whether two string should be compared as case sensitive or not|yes|
+
+
+### Filtering with Extraction Functions
```

Comment:
@gianm Added a link in each individual filter section pointing to "Filtering with Extraction Functions"